app/Http/Controllers/ReviewsController.php

```diff
@@ -35,6 +35,7 @@ public function __construct(PostManager $post)
 
         $this->middleware('auth:api');
         $this->middleware('role:admin');
+        $this->middleware('role:staff');
```

Comment:
I think these should be combined - this will check first "is this user an admin", and then "is this user staff"...

```php
$this->middleware('role:admin,staff');
```

Reply:
ah yes that is helpful bc the tests were not passing bc it thought it needed _both_ roles, `admin` AND `staff`!